CorsixTH/Lua/app.lua

```diff
@@ -1517,6 +1517,26 @@ function App:quickLoad()
   end
 end
 
+--! Function to check the savegame version matches or is older than the installation of CorsixTH
+--!param save_version (num)
+--!param save_release (string) Game's human readable game version
+--!return true if compatible, otherwise false
+function App:checkCompatibility(save_version, save_release)
+  local app_version = self.savegame_version
+  local version_name = self:getVersion(app_version)
+  local save_release_ver = save_release or "Trunk" -- For compatibility
+  local app_version_humanize = tostring(version_name .. " [" .. app_version .. "]")
+  local save_version_humanize = tostring(save_release_ver .. " [" .. save_version .. "]")
```

Comment:
Do one string at a time, line 1527 now switches to another version text, and 1528 switches back again to the previous conversion. Intuitively, you read it as 1 sequence of assignments, (`b = ..a .. ; c = .. b ..; d = ..c..;` etc) but it's not.

Use an empty line between them to show the reader how to read it.

Can't you fold the string concatenation stuff at 1528 and 1529 into the `_S.errors.compatibility_error` ? (would only work if it's a new string I guess)